features/dataset-view.feature

```diff
@@ -330,6 +330,46 @@ Feature: a user visit the dataset page
 		| pre_03AUG2015_update 								| folder containing originally submitted data files, prior to update Aug 3rd 2015.	| Directory 		| 50.00 MiB 	|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/pre_03AUG2015_update |
 		| readme.txt 										|				                                                                    | Readme 			| 337 B 		|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/readme.txt |
 
+	@ok @javascript @pr521
+	Scenario: To see the citation box and and the citation source
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I should see a button input "Cite Dataset"
+		When I press "Cite Dataset"
+		Then I should see "Text"
+		And I should see "RIS"
+		And I should see "BibTeX"
+
+	@ok @javascript @pr521
+	Scenario: To get Text file after clicking the Text button in the citation box
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I press "Cite Dataset"
+		When I click on the "Text" button
+		Then I go to "http://data.datacite.org/text/x-bibliography/10.5524/101001"
+		And I should see "Genome sequence of the duck (Anas platyrhynchos)"
+
+	@ok @javascript @pr521
+	Scenario: To get RIS file after clicking the RIS in the citation box
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I press "Cite Dataset"
+		When I click on the "RIS" button
+		Then the response should contain "101001"
+		# 200: the request completed successfully
+		And the response should contain "200"
+
+	@ok @javascript @pr521
+	Scenario: To get BibTeX file after clicking the BibTeX in the citation box
+		Given I am not logged in to Gigadb web site
+		And I am on "/dataset/101001"
+		And I press "Cite Dataset"
+		When I click on the "BibTeX" button
+		Then the response should contain "101001"
+		# 200: the request completed successfully
+		And the response should contain "200"
```

Comment:
Maybe a String longer than "200" and "101001" can be checked for a more accurate test? For example, check for "200 OK".

Reply:
However, for example, here is the network output of my chrome browser:
![image](https://user-images.githubusercontent.com/64770635/101868394-338f9080-3bb8-11eb-9f57-4d35ecfe7464.png)
If a longer string was appended, the test would fail.